api/src/test/java/com/epam/pipeline/test/web/AbstractControllerTest.java

```diff
@@ -197,8 +199,11 @@ public void performRequestWithoutResponse(final MockHttpServletRequestBuilder re
     public MultiValueMap<String, String> multiValueMapOf(Object... objects) {
         final MultiValueMap<String, String> map = new LinkedMultiValueMap<>();
         for (int i = 0; i < objects.length; i += 2) {
-            map.add(String.valueOf(objects[i]).replaceAll("[\\[\\]]", ""),
-                    String.valueOf(objects[i + 1]).replaceAll("[\\[\\]]", ""));
+            if (objects[i + 1] instanceof List) {
+                map.add(String.valueOf(objects[i]), String.valueOf(objects[i + 1]).replaceAll("[\\[\\]]", ""));
```

Comment:
I suppose it's cleaner to use `String.valueOf` for each of the list elements and just join them after using f.e. `Collectors.joining`.